components/script/dom/htmlinputelement.rs

```diff
@@ -408,6 +408,17 @@ impl TextControl for HTMLInputElement {
     fn textinput(&self) -> &DomRefCell<TextInput<ScriptToConstellationChan>> {
         &self.textinput
     }
+
+    fn selection_api_applies(&self) -> bool {
```

Comment:
Can you link to the relevant section of the spec that defines this behaviour?

Reply:
Yep